lib/maplibre-compose-material3/src/desktopMain/kotlin/dev/sargunv/maplibrecompose/material3/util.desktop.kt

```diff
@@ -0,0 +1,5 @@
+package dev.sargunv.maplibrecompose.material3
+
+import dev.sargunv.maplibrecompose.material3.controls.ScaleBarMeasure
+
+internal actual fun scaleBareMeasurePreference(): ScaleBarMeasure? = null
```

Comment:
should put a TODO comment here; macOS at least probably has an API for this identical to the iOS one. unsure about other platforms.

Reply:
I also think there is an API for that on MacOS. But in the `build.gradle`, the `desktop` target is actually the `jvm` target, which is disjunct from the macOs target according to this illustration:

![default-hierarchy-example](https://github.com/user-attachments/assets/d0817059-1343-4382-8f43-dd3aa964b140)

(source: https://kotlinlang.org/docs/multiplatform-hierarchy.html#default-hierarchy-template )

Reply:
Yeah, we'd have to detect platform and call in to native APIs probably (if there's not already a wrapper library for it). Can just leave it as a TODO for now (I'll take care of it later)

Reply:
note for later, [this](https://javadoc.lwjgl.org/org/lwjgl/system/macosx/ObjCRuntime.html) is a helper to make calls to ObjC from the JVM